src/main/java/Sidtacphi.java

```diff
@@ -0,0 +1,255 @@
+                    throw new SidInvalidFormatException("Please input a name for your Event" 
+                    + "task, along with a start and end time.");
+                } else if (input.charAt(5) == ' ') {
+                    inputArgs = input.substring(6).split("\\s*/from\\s*");
+                    if (inputArgs.length == 2) { 
+                        String[] startAndEnd = inputArgs[1].split("\\s*/to\\s*");
+                        if (startAndEnd.length == 2) {
+                            taskList.add(new Event(inputArgs[0], startAndEnd[0], startAndEnd[1]));
+                        } else {
+                            throw new SidInvalidFormatException("Please put in the starting and ending time " 
+                            + "using \"/from <time>\" followed by \"/to <time>\" for Event tasks.");
+                        }
+                    } else {
+                        throw new SidInvalidFormatException("Please put in the starting and ending time " 
+                        + "using \"/from <time>\" followed by \"/to <time>\" for Event tasks.");
+                    }
+                } else {
+                    throw new SidException("\"" + input + "\" is not a valid command.");
+                }
+                break;
+            case DEADLINE:
+                if (input.length() < 9) {
+                    throw new SidInvalidFormatException("Please input a name for your Event" 
+                    + "task, along with a start and end time.");
+                } else if (input.charAt(8) == ' ') {
+                    inputArgs = input.substring(9).split("\\s*/by\\s*");
+                    if (inputArgs.length == 2) { 
+                        taskList.add(new Deadline(inputArgs[0], inputArgs[1]));
+                    } else if (inputArgs.length == 1) {
+                        throw new SidInvalidFormatException("Please write in the deadline" 
+                        + "using \"/by <time>\" for Deadline tasks. ");
+                    } else {
+                        throw new SidInvalidFormatException("Please do not write in more than 1 deadline. ");
+                    }
+                } else {
+                    throw new SidException("\"" + input + "\" is not a valid command.");
+                }
+                break;
+        }
+        
+        System.out.println("\nSidtacphi: I have added \"" + taskList.get(taskList.size() - 1) + "\".");
+        
+        if (taskList.size() == 1) {
+            System.out.println("Sidtacphi: You now have 1 task in your list.");
+        } else {
+            System.out.println("Sidtacphi: You now have " + taskList.size() + " tasks in your list.");
+        }
+    }
+
+    /**
+     * Marks/Unmarks the task given.
+     * 
+     * @param toMark to mark the task as done when true, and to unmark when false
+     * @param input 
+     */
+    private static void markTaskAs(boolean toMark, String input) throws SidException {
```

Comment:
toMark is a boolean variable name so perhaps you want to make it sound like a boolean, (ie. it sounds like it can be responded with either a true or false) so for example maybe you can name it as "hasBeenMarked"